opslevel/datasource_opslevel_scorecards.go

```diff
@@ -0,0 +1,105 @@
+package opslevel
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opslevel/opslevel-go/v2023"
+)
+
+func datasourceScorecards() *schema.Resource {
+	return &schema.Resource{
+		Read: wrap(datasourceScorecardsRead),
+		Schema: map[string]*schema.Schema{
+			"ids": {
```

Comment:
if you look at some of the other "plural" datasources - i think we provide a way to "filter down" the scorecards using certain fields - IE "give me all the scorecards where the owner == X"

This "filter" would be optional but could be provided by the customer to scope the overall list scorecards returned.

Reply:
@rocktavious I noticed that for the other data sources (e.g. repository or filter) it allows filtering based on a criteria that is available in the API. [Repository data source code](https://github.com/OpsLevel/terraform-provider-opslevel/blob/main/opslevel/datasource_opslevel_repositories.go#L41)

For example, with Repository you can filter by `tierAlias`, which is defined in gql:

```
query {
  account {
    repositories(tierAlias: "tier_3") {
      nodes {
        id
        name
      }
    }
  }
}
```

However for Scorecards we don't have any criteria like this we can filter by in the API. 

We could implement filtering by `ownerId` or `filterId` but it would have to be done on the client side either in opslevel-go or the terraform provider. Should we do this? I think it would be better for the API change to be made first, then we can make the changes in tools.

Reply:
you are correct - we'll need to make a API feature request - for now this is fine then